app/models/katello/content_view_version.rb

```diff
@@ -298,7 +298,8 @@ def docker_tag_count
     end
 
     def docker_tags
```

Comment:
Can you modify the test here so that it would fail without your change?

https://github.com/Katello/katello/blob/c5116b3e40fc6dcfd569cb0bd81207f8dbaf7195/test/models/content_view_version_test.rb#L94

Reply:
I've added a part to the test where an un-archived repo with a docker_meta_tag is introduced.  This returns a count of 2 rather than one if looking at `repositories.docker_type` rather than `repositories.archived.docker_type`.

Reply:
I've added a part to the test where an un-archived repo with a docker_meta_tag is introduced. This returns a count of 2 rather than one if looking at repositories.docker_type rather than repositories.archived.docker_type.